File: sheet/prog/arg.jl
#!/usr/bin/env julia

# juliaの実行ファイルやソースコードは含まない [[注意]]
# julia ./arg.jl Hello => ["Hello"]
println(ARGS)

# ソースコードは
println(PROGRAM_FILE)

# 標準ではパースの方法はない

File: sheet/prog/arg.r

commandArgs()
# [1] "/(略)/bin/R"
# [2] "--vanilla"
# [3] "--args"
# [4] "hello"

commandArgs(trailingOnly=T)
# [1] "hello"

# R --vanilla --args hello < arg.r
# Rscript arg.r hello
